src/refiners/fluxion/utils.py

```diff
@@ -34,6 +34,31 @@ def interpolate(x: Tensor, factor: float | torch.Size, mode: str = "nearest") ->
     )  # type: ignore
 
 
+# Adapted from https://github.com/pytorch/vision/blob/main/torchvision/transforms/_functional_tensor.py
+def normalize(tensor: Tensor, mean: list[float], std: list[float], inplace: bool = False) -> Tensor:
```

Comment:
maybe we can jaxtype annotate  (since it is in utils there is an ambiguity: this is a specific to image data